src/data/zmod/quotient.lean

```diff
@@ -132,4 +132,8 @@ rfl
   minimal_period ((•) a) b = fintype.card (orbit (zpowers a) b) :=
 by rw [←fintype.of_equiv_card (orbit_zpowers_equiv a b), zmod.card]
 
+@[to_additive] instance minimal_period_pos [fintype (orbit (zpowers a) b)] :
```

Comment:
I'm not sure we want to turn such a specific proposition into a typeclass.

How about making this just a plain `def` rather than an `instance` of `fact`?

Reply:
See for example https://leanprover-community.github.io/mathlib_docs/notes.html#fact%20non-instances

Reply:
See also @ericrbg 's remark [on Zulip](https://leanprover.zulipchat.com/#narrow/stream/144837-PR-reviews/topic/.2314201.20minimal.20period.20is.20positive/near/282906663) How about using https://leanprover-community.github.io/mathlib_docs/algebra/ne_zero.html#ne_zero ?

Reply:
Sounds good. The main thing I want is a `fintype zmod` instance, so I'll just add a `ne_zero → fintype zmod` instance, which should be fine.